examples/kubernetes-non-vhd-distros.json

```diff
@@ -0,0 +1,64 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v3",
+      "distro": "ubuntu"
+    },
+    "agentPoolProfiles": [

```

Comment:
@marosset FYI, moved the gen2 E2E coverage to this api model, which is used by the PR E2E spec. So now all PRs will validate these 4 non-VHD flavors.